ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-1.9-release-4.18.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+releases:
+  initial:
+    integration:
+      name: "4.18"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.18"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 2Gi
+tests:
+- as: peerpods-e2e-azure
+  cron: '@yearly'
+  steps:
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      ENABLEGPU: "false"
+      ENABLEPEERPODS: "true"
+      INSTALL_KATA_RPM: "false"
+      OPERATOR_INDEX_VERSION: 1.9.0
+      OPERATOR_UPDATE_CHANNEL: stable
+      TEST_FILTERS: ~VMonly&;~00160&;~00143&;~00142&;~00081&;~00126&;~00132&;~00094&;~00332&;~43221&;~422081&;~00121&;~00091&;~00170&
```

Comment:
Change this to 
```
TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&;41566
TEST_SCENARIOS: Kata
```
to avoid the race condition of multiple tests.
41566 will run by itself, install OSC and run a pod

Reply:
Changed, thanks!